src/types/atproto/starterPack.ts

```diff
@@ -0,0 +1,21 @@
+import {AppBskyGraphDefs} from '@atproto/api'
+
+export {
+  /**
+   * Renamed to clarify source of this util, but directly aliases the original.
+   * {@link AppBskyGraphDefs.isStarterPackViewBasic}
+   */
+  isStarterPackViewBasic as isBasicView,
+  /**
+   * Renamed to clarify source of this util, but directly aliases the original.
+   * {@link AppBskyGraphDefs.isStarterPackView}
+   */
+  isStarterPackView as isView,
+} from '@atproto/api/dist/client/types/app/bsky/graph/defs'
```

Comment:
Relying on the internal folder structure of a dependency is not ideal. Any reason why you are not doing the following instead ?

```suggestion
export const isBasicView = AppBskyGraphDefs.isStarterPackViewBasic
export const isView = AppBskyGraphDefs.isStarterPackView
```